cli/cmd/cloud_account_test.go

```diff
@@ -0,0 +1,183 @@
+package cmd
+
+import (
+	"encoding/json"
+	"strings"
+	"testing"
+
+	"github.com/lacework/go-sdk/api"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDetailsTable(t *testing.T) {
```

Comment:
Niiiiiiiiiiiice!!! I wish I could see more tests like this one!!!!

![tenor-250560347](https://user-images.githubusercontent.com/5712253/189813258-3c1de670-021e-43dc-aee5-492ed974ee33.gif)